helm/cluster-eks/values.schema.json

```diff
@@ -13,6 +13,10 @@
             "type": "object",
             "title": "Node pool",
             "properties": {
+                "amiType": {
+                    "type": "string",
+                    "title": "AMI type"
```

Comment:
nitpick: would it make sense to have an `enum` with available ami types or would it be too hard to maintain?

Reply:
I thought about that and decided against it due to maintainability, but on second thoughts the list is quite small. the current options are:

```
    AL2_x86_64
    AL2_x86_64_GPU
    AL2_ARM_64
    AL2023_x86_64_STANDARD
    AL2023_ARM_64_STANDARD
    CUSTOM
```

All the `AL2_` options are deprecated so that leaves only 3 to maintain. i don't think maintaining this would be too much of a burden, they're not going to change often.